ParisBNF/abb/BNFabb80.xml

```diff
@@ -96,84 +101,95 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </msItem>
 
                             <msItem xml:id="ms_i1.10">
-                                <locus from="39r" to="51v"/>
-                                <title type="complete" ref="LIT6443Sunday6"/>
+                                <locus from="39r" to="39v"/>
+                                <title type="incomplete" ref="LIT6443Sunday6"/>
```

Comment:
It can stay like this, but it is always helpful for the readers to have a short `<note>` in which the character of the incompleteness is indicated.